modules/cluster-logging-log-forwarding-configure.adoc

```diff
@@ -0,0 +1,134 @@
+= Configuring log forwarding
+
+To configure log forwarding, edit the Cluster Logging Custom Resource (CR) to enable forwarding and create a Log Forwarding Custom Resource to specify the outputs, pipelines, and enable log forwarding. 
+
+[NOTE]
+====
+Enabling the log forwarding feature disables the xref:../../logging/config/cluster-logging-external.adoc#cluster-logging-external[default log forwarding feature].
+====
+
+If you enable log forwarding, any undefined source types will be dropped. For example, if you specified a pipeline for the `logs.app` 
+source type, but did not specify a pipeline for the `logs.infra`type, `logs.infra` are dropped.
+
+.Procedure
+
+. Edit the Cluster Logging Custom Resource (CR) in the `openshift-logging` project: 
++
+----
+$ oc edit ClusterLogging instance
+----
+
+. Set `logforwardingtechpreview` to `enabled`.
++
+[source,yaml]
+----
+spec:
+  clusterlogging.openshift.io/logforwardingtechpreview: enabled <1>
+----
+<1> Enables and disables log forwarding. Set to `enable` to use log forwarding. Set to `disabled` to use the {product-title} Elasticsearch logstore.
+
+. Create the Log Forwarding instance:
++
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1alpha1"
+kind: "LogForwarding"
+spec:
+  disableDefaultForwarding: true <1>
+  outputs:  <2>
+   - type: “elasticsearch”
+     name: elasticsearch
+     endpoint: elasticsearch.svc.messaging.cluster.local
+     secrets:
+        name: elasticsearch
+   - type: “forward”
+     name: secureforward-offcluster
+     endpoint: https://secureforward.offcluster.com:9200
+     insecure: true
+  pipelines: <3>
+   - name: container-logs
+     inputSource: logs.app
+     outputRefs:
+     - elasticsearch
+     - secureforward-offcluster
+----
+<1> Set to `true` to enable log forwarding.
+<2> Add one or more endpoints:
+* Specify the type of output, either `elasticseach` or `forward`.
+* Enter a name for the output.
+* Enter the endpoint, either the server name or FQDN.
+* Optionally, enter the name of the secret required by the endpoint for TLS communication. The secret must exist in the openshift-logging project.
+* Specify insecure: true if the endpoint does not use a secret, resulting in insecure communication.
+<3> Add one or more pipelines:
+* Enter a name for the pipeline
+* Specify the source type, logs.app, logs.infra, or logs.audit.
+* Specify the name of one or outputs configured in the CR.
+ 
+A typical log forwarding configuration would be similar to the following examples:
+
+.Sample resource for an `elasticsearch` custom resource
+
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1alpha1
+kind: LogForwarding
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+    - name: user-created-es
+      type: elasticsearch
+      endpoint: 'elasticsearch-server.openshift-logging.svc:9200'
+      secret:
+        name: piplinesecret
+  pipelines:
+    - name: clo-default-app-pipeline
+      inputType: logs.app
```

Comment:
please change to `inputSource`.  I'll update the code and other refs accordingly